fixtures/typescript-css-modules/src/render.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { renderToString } from 'react-dom/server';
 import dedent from 'dedent';
 
-import { Render } from '../../../../../sku-types';
+import { Render } from 'sku';
```

Comment:
Did the type-only import rules not get promoted to the general eslint config? Would have expected this to be:
```suggestion
import type { Render } from 'sku';
```

Reply:
`eslint-config-seek` doesn't actually enforce this rule. Probably worth updating, I've building a list of eslint rules I think we should add to it, I'm sure Remus has a few more too.